.github/workflows/notify_weekly.yaml

```
@@ -13,15 +13,13 @@ jobs:
     - name: Create Mattermost Message
       run: |
         DATE=$(date --date="7 days ago" +"%Y-%m-%d")
-        echo "MERGED_URL=https://github.com/pulls?q=merged%3A%3E${DATE}+org%3Acertbot" >> $GITHUB_ENV
-        echo "UPDATED_URL=https://github.com/pulls?q=updated%3A%3E${DATE}+org%3Acertbot" >> $GITHUB_ENV
+        echo "ASSIGNED_PRS=https://github.com/certbot/certbot/pulls?q=is%3Apr+is%3Aopen+updated%3A%3E%3D${DATE}+assignee%3A*" >> $GITHUB_ENV
```

what if instead of this URL we used something like

```suggestion
        echo "ASSIGNED_PRS=https://github.com/pulls?q=is%3Apr+is%3Aopen+updated%3A%3E%3D${DATE}+assignee%3A*+user%3Acertbot" >> $GITHUB_ENV
```

the benefit here is it shows PRs on all repos under the certbot org rather than just the main repo. the downside is you have to be logged into github for that URL to work

whatcha think?

---

I think that is okay considering everyone looking at this, is likely logged into Github anyway